src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -913,6 +913,11 @@ public AppPage verifyContains(String searchString) {
         return this;
     }
 
+    public void verifyContainElement(By by) {
```

Comment:
@boxin-yang still need one last change here before we set it to merge, this method doesn't seem to be used